tests/e2e/streams/streams_tests.py

```diff
@@ -143,23 +157,27 @@ def test_check_stream(producer, topics, connection, transformation):
         producer.send(topics[0], message).get(timeout=60)
 
     def check_check_stream(batch_limit):
-        assert transformation == "transform.simple" \
+        assert (
+            transformation == "transform.simple"
             or transformation == "transform.with_parameters"
+        )
```

Comment:
```suggestion
        assert transformation in TRANSFORMATIONS_TO_CHECK
```

Reply:
I think this is my bad... I think I put this assert here to make the if statement below simpler. But I think this was a big mistake from my side, because it doesn't express the intent well enough. If you want to improve this, then I would remove this assert completely and change the if statement below to:
```python
    if transformation == "transformation.simple":
        # ....
    elif transformation == "transformation.with_parameters":
        # ....
    else:
        assert False
```
This way the intention is much clear.